core/src/main/resources/jenkins/agents/CloudSet/index.jelly

```diff
@@ -39,36 +45,45 @@ THE SOFTWARE.
               </a>
             </l:isAdmin>
           </l:app-bar>
-          <table id="clouds" class="jenkins-table sortable">
-            <thead>
-              <tr>
-                <th class="jenkins-table__cell--tight"/>
-                <th initialSortDir="down">${%Name}</th>
-                <th class="jenkins-table__cell--tight"/>
-              </tr>
-            </thead>
-            <tbody>
-              <j:forEach var="cloud" items="${it.clouds}">
-                <tr id="cloud_${cloud.name}">
-                  <td data="${cloud.icon}" class="jenkins-table__cell--tight jenkins-table__icon">
-                    <div class="jenkins-table__cell__button-wrapper">
-                      <l:icon src="${cloud.iconClassName}" tooltip="${cloud.iconAltText}"/>
-                    </div>
-                  </td>
-                  <td>
-                    <a href="${it.getCloudUrl(request,app,cloud)}" class="jenkins-table__link model-link inside">${cloud.name}</a>
-                  </td>
-                  <td class="jenkins-table__cell--tight">
-                    <div class="jenkins-table__cell__button-wrapper">
-                      <a href="${it.getCloudUrl(request,app,cloud)}configure" class="jenkins-table__button">
-                        <l:icon src="symbol-settings"/>
-                      </a>
-                    </div>
-                  </td>
+          <f:form method="post" name="config" action="reorder">
+            <table id="clouds" class="jenkins-table">
+              <thead>
+                <tr>
+                  <th tooltip="${%Drag and drop cells to reorder}">${%Order}</th>
+                  <th>${%Name}</th>
+                  <th/>
                 </tr>
-              </j:forEach>
-            </tbody>
-          </table>
+              </thead>
+              <tbody class="with-drag-drop">
```

Comment:
New class name prefixed neither `app` nor `jenkins`, isn't this something we should no longer do?

Reply:
It's not a new class https://github.com/search?q=repo%3Ajenkinsci%2Fjenkins%20with-drag-drop&type=code